raven/src/main/java/com/getsentry/raven/connection/AsyncConnection.java

```diff
@@ -84,7 +81,7 @@ public AsyncConnection(Connection actualConnection, ExecutorService executorServ
      * @param gracefulShutdown Indicates whether or not the shutdown operation should be managed by a ShutdownHook.
      */
     public AsyncConnection(Connection actualConnection, ExecutorService executorService, boolean gracefulShutdown) {
-        this(actualConnection, executorService, gracefulShutdown, DEFAULT_SHUTDOWN_TIMEOUT);
+        this(actualConnection, executorService, gracefulShutdown, DefaultRavenFactory.ASYNC_SHUTDOWN_TIMEOUT_DEFAULT);
```

Comment:
It seems a little weird to grab these out of the default factory. It seems like we should either:

1.) Define the default values here, and fill them in if they either not provided to this constructor or are null in the other constructor, or,
2.) Define the default values in the default factory, and drop this shorter constructor.

(This same comment applies to the similar constructors below.)